jujugui/options.py

```diff
@@ -22,14 +22,14 @@ def update(settings):
     _update(settings, 'jujugui.api_path', default=DEFAULT_API_PATH)
     _update(settings, 'jujugui.ga_key', default='')
     _update(settings, 'jujugui.password', default=None)
-    _update(settings, 'jujugui.socket_path', default=None)
     _update(settings, 'jujugui.user', default='')
+    _update(
+        settings, 'jujugui.socketTemplate', default='/environment/$uuid/api')
```

Comment:
Added documentation for socketTemplate to HACKING.rst (along with raw and combine) as those are things that can be set by the user to alter behavior. I didn't document apiAddress as it's always going to be set by whatever software is deploying the gui; if you feel strongly though I'm happy to add a section to HACKING.rst about those as well.